metastats-domain/src/main/groovy/life/qbic/metastats/MetaStatsLogManager.groovy

```diff
@@ -0,0 +1,36 @@
+package life.qbic.metastats
+
+/**
+ * <class short description - One Line!>
```

Comment:
Insert short description
```suggestion
Class responsible for correct error handling and logging 
```

Reply:
To keep the formatting (* is missing) I added your suggestions manually: a9fd7aa

Reply:
you can also make a suggestion upon a suggestion and commit that